backend/Readme.md

```diff
@@ -20,130 +20,153 @@
-  export PGPASSWORD=admin
-  export PGDATABASE=postgres
+  python raw_backend --insert --replication --source https://download.geofabrik.de/europe/turkey-latest.osm.pbf --country 127
   ```
 
-  Choose your source for the Import
-
-  > You can Download Planet pbf file [Here](https://planet.osm.org/pbf/) or Use Geofabrik Pbf file [Here](https://osm-internal.download.geofabrik.de/index.html) with full metadata (Tested with .pbf file) , or pass download link to script itself . Follow -h help
-
-  - Test with the sample data
-
-    ```
-    python raw_backend --insert
-    ```
-
-    > This will insert the sample data provided on the code , You can check your backend setup with this
-
-  - For eg : To import Turkey and Enable replication later on
-
-    ```
-    python raw_backend --insert --replication --source https://download.geofabrik.de/europe/turkey-latest.osm.pbf --country 127
-    ```
-
-    > Here Insert option will do the import after downloading osm.pbf file from source , You can supply filepath of already downloaded file to --source as well . Replication will enable and Prepare the tables for replication and start the replication right away until data becomes now , Country parameter is read from fid of /backend/countries which will make sure to only maintains the replication data for that country
-
-  
-  - Import your osm file and run replication for only your custom geojson 
-
-    ```
-      python raw_backend --replication --boundary pokhara.geojson
-    ```
-    Example of geojson : 
-      ```
-          {
-          "type": "Feature",
-          "properties": {},
-          "geometry": {
-            "coordinates": [
-              [
-                [
-                  83.90895770965955,
-                  28.279633888511327
-                ],
-                [
-                  83.90895770965955,
-                  28.131383546395526
-                ],
-                [
-                  84.10168708213502,
-                  28.131383546395526
-                ],
-                [
-                  84.10168708213502,
-                  28.279633888511327
-                ],
-                [
-                  83.90895770965955,
-                  28.279633888511327
-                ]
-              ]
-            ],
-            "type": "Polygon"
-          }
-        }
-      ```
+  > Here Insert option will do the import after downloading osm.pbf file from source , You can supply filepath of already downloaded file to --source as well . Replication will enable and Prepare the tables for replication and start the replication right away until data becomes now , Country parameter is read from fid of /backend/countries which will make sure to only maintains the replication data for that country
+
+- Import your osm file and run replication for only your custom geojson
 
+  ```
+    python raw_backend --replication --boundary pokhara.geojson
+  ```
 
-  To Run Replication only
+  Example of geojson :
```

Comment:
From here on All the installation instructions are moved to code block . Do verify that and make sure it is readable. You can preview your file and you will notice something like this 
<img width="1206" alt="image" src="https://github.com/hotosm/raw-data-api/assets/36752999/891f2f91-10e2-4974-a128-d48a314936ce">